mono/metadata/locales.h

```diff
@@ -79,9 +79,7 @@ int ves_icall_System_Globalization_CompareInfo_internal_index (MonoCompareInfo *
 ICALL_EXPORT
 int ves_icall_System_Globalization_CompareInfo_internal_index_char (MonoCompareInfo *this_obj, MonoString *source, gint32 sindex, gint32 count, gunichar2 value, gint32 options, MonoBoolean first);
 
-ICALL_EXPORT
-int
-ves_icall_System_Threading_Thread_current_lcid (void);
+#define MONO_LOCALE_INVARIANT (0x007F)
```

Comment:
Alternatively if we want to maintain more of the illusion:
`#define mono_thread_current_lcid() ((int)0x007F/* Invariant */)`